Frequently_used_code/ARD_overpass_predictor.ipynb

```diff
@@ -0,0 +1,864 @@
+{
```

Comment:
<p>These commented out bits make the code a little harder to follow:</p><p> </p><pre class="ql-syntax" spellcheck="false">#Sentinel_2A = Sentinel_2A + datetime.timedelta(hours=10) #convert to local time (Aus eastern standard time) = utc + 10 hours
#Sentinel_2A&nbsp;### to AEDT, add 11h not 10 ###
</pre><p> </p><p>I think it might be better to simplify the code by removing them from here and instead make it really clear in the <code>Combine dataset</code>  bit that the times are in UTC (?) (and possibly give an example of how to convert the columns in the final table there). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>